packages/kbn-test/src/functional_tests/run_tests/run_tests.ts

```diff
@@ -113,6 +114,12 @@ export async function runTests(log: ToolingLog, options: RunTestsOptions) {
             return;
           }
 
+          if (serverless) {
+            // A dirty temporary solution to wait for ES to process Kibana privileges.
+            // Stateless ES takes 30-35 seconds to process privileges request sent by Kibana
+            await scheduler.wait(35000);
```

Comment:
Have we looked into any alternatives with the elasticsearch team?  @afharo do you know who we can reach out to?

35 seconds per suite is going to add a not-insignificant amount of minutes to each build.  I'm hoping we can find a quicker check.

Reply:
@elastic/appex-qa is working close with Kibana security folks @azasypkin @jeramysoucy to enable roles testing. Our approach is to add a FTR service that will generate cookie for a specific user role, that will be used for browser authentication and kbnClient init. We expect it to work both with Kibana CI and MKI. 

This PR is making quite many changes in FTR and doesn't match the concept we agreed on, I suggest @maximpn to join the sync meeting `FTR with Serverless Security Sync` this Thursday to discuss approach with all the folks involved.

Reply:
@dmlemeshko, @maximpn is on PTO during this week.

Reply:
> @afharo do you know who we can reach out to?

TBH, I just asked in the general `#elasticssearch` Slack channel.

---

@maximpn, did your test include [these changes](https://github.com/elastic/kibana/pull/169926)? I wonder if the need for the delay occurred before I managed to merge my PR.